kuma/core/middleware.py

```diff
@@ -53,7 +54,7 @@ def process_request(self, request):
             return response
 
         request.path_info = '/' + prefixer.shortened_path
-        request.locale = prefixer.locale
+        request.locale = prefixer.locale or settings.LANGUAGE_CODE
```

Comment:
I'm not sure about this, to be honest. We shouldn't break URLs or force a rendering of URLs with locale that are supposed to ignore the locale in their path. What's the worst that could happen with this change?